### DISCUSSION TITLE
4.3 Installation problem

### DISCUSSION THREAD

**Discussion by nicolassavard** [udotdevelopment/ATSPM#161] [Q&A] [answered]
I'm unable to install version 4.3... The web page is not loading, see the attached file. Can someone help me thanks!

![2022-08-26 15_31_02-serveur 2019_01  En fonction  - Oracle VM VirtualBox](https://user-images.githubusercontent.com/106619965/186978703-2ddd0016-0877-421f-b2f0-1588503c8df7.png)

**Comment by jamesbley-rhythm** [comment, marked as answer]
I can confirm that I have the same issue. New Windows Server 2012 R2 Datacenter install. I also got a framework 4.7.2 error initially which was resolved by installing that version (not mentioned in the installation guide). I believe the web.config file might be bad.

The message "an attempt was made to load a program with an incorrect format" typically means that 32-bit binaries were attempted to be loaded within IIS 7 on a 64-bit machine. I'm going to try something and will report back here with the results.

**Reply by jamesbley-rhythm** [reply]
That was it. For the IIS Application Pools, set "Enable 32-Bit Applications" to True. I did this for all 3 (.NET v4.5, .NET v4.5 Classic, and DefaultAppPool) even though I was only using .NET v4.5.

Right-click on the Application Pool (.NET v4.5 for instance) and select Advanced Settings.

![image](https://user-images.githubusercontent.com/36171783/189119626-f870bff6-e21b-410b-92a7-9de167fb50b0.png)

**Reply by jamesbley-rhythm** [reply]
Now I get the error about the dependency on the 'TotalDelay' column that someone already reported. Hopefully there's a fix available for that soon.
